frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -46,18 +48,24 @@ export function removeLeakableResource(leakedResources: Set<string>, resource) {
 }
 
 export function createResource(resource) {
-  execSync(`echo '${JSON.stringify(resource)}' | kubectl create -f -`);
+  execSync(`echo '${JSON.stringify(resource)}' | kubectl apply -f -`);
 }
 
 export function createResources(resources) {
   resources.forEach(createResource);
 }
 
-export function deleteResource(resource) {
+export function deleteResource(resource, ignoreNotFound = true) {
   const kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
-  execSync(
-    `kubectl delete -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`,
-  );
+  if (ignoreNotFound) {
+    execSync(
+      `kubectl delete --ignore-not-found=true -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`,
```

Comment:
```suggestion
      `kubectl delete --ignore-not-found=${ignoreNotFound} -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`,
```

Reply:
It seems better to don't add `if esle` in this way, changed to use `--ignore-not-found=${ignoreNotFound}` and not use `if else`.